ui/narrative/methods/run_stringtie/spec.json

```diff
@@ -139,10 +116,6 @@
             "input_parameter" : "expression_set_suffix",
             "target_property" : "expression_set_suffix"
         },
-        {
-          "input_parameter" : "novel_isoforms",
-          "target_property" : "novel_isoforms"
```

Comment:
Will this break things?  Do we need a hidden parameter that sets these parameters to False or Null?

Reply:
Tian thinks this should be fine.  Let's find out.